.github/pull_request_template.md

```diff
@@ -0,0 +1,11 @@
+### ☑️ Resolves
+
+* Fix #…
```

Comment:
```suggestion
Fix #…
```

not a list?

Reply:
Lists render the issues then with the nice readable headline instead of just the issue number:
![image](https://user-images.githubusercontent.com/213943/233092005-5e17cfec-6420-46a3-9ed3-6327a35d0a82.png)